services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/taydennys/TaydennysAuthorizer.kt

```diff
@@ -26,4 +29,18 @@ class TaydennysAuthorizer(
             hakemusAuthorizer.authorizeHakemusId(it, permissionCode)
         }
     }
+
+    @Transactional(readOnly = true)
+    fun authorizeAttachment(
```

Comment:
There are no tests for this.

Reply:
I added a test for this and also added more tests for this in the controller.